### PR TITLE
feat/CED-173: Better benchmarking

### PR DESCRIPTION
**Pull request by nravic**
Encapsulates a bunch of changes. 

Added: 
- A working directory to specify to execTask 
- get out of go benchmarking tooling and into python (ugh)
- replace go benchmarking with running a pprof server now instead to profile checkpoint/restore. helps segment different ops 
- some coarse data generated from benchmark.py 

Todo (will be further tickets) 
- Fix containerization 
- push to bq or datastore for more analysis 
- analyze generated pprof files using the pprof proto

### CODE REVIEW

File: api/server.go [hunk 1]
```diff
@@ -345,9 +344,22 @@ func (s *service) runTask(task string) (int32, error) {
...
+	if workingDir != "" {
+		cmd.Dir = workingDir
+	}
+
 	cmd.Stdin = nullFile
-	cmd.Stdout = w
-	cmd.Stderr = w
+	if logOutputFile == "" {
+		// default to /var/log/cedana-output.log
+		logOutputFile = "/var/log/cedana-output.log"
```
Comment (rst0git) [inline api/server.go:354]: It might be better to use constants for default values

File: cmd/client-daemon.go [hunk 2]
```diff
@@ -80,7 +85,7 @@ var stopDaemonCmd = &cobra.Command{
 	RunE: func(cmd *cobra.Command, args []string) error {
 		// kill -9 daemon
 		// read from PID file
-		pidFile, err := os.ReadFile("/tmp/cedana.pid")
+		pidFile, err := os.ReadFile("/var/log/cedana.pid")
```
Comment (rst0git) [inline cmd/client-daemon.go:88]: `/run/cedana.pid` might be more appropriate location for the pid file

Comment (nravic) [reply cmd/client-daemon.go:88]: good call! fixed

**Review by bsmithai — APPROVED** [review]
lgtm

**Pull request merged**